### DISCUSSION TITLE
How to show multiple items per page with Pagination component?

### DISCUSSION THREAD

**Discussion by idafoh** [mantinedev/mantine#2600] [Q&A] [answered]
I want to show 10 items per page total 100 items but Pagination component always shows me 1 item per page and there 100 pages what I want is 10 item per page total 10 pages. How could we do that (`itemsPerPage` prop but I cannot find it)?

Is it my misunderstanding (sorry about that) or is there something missing?

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/beautiful-star-o0wcx?file=/src/App.tsx

**Reply by idafoh** [reply]
Thanks for response. But I think creating async pagination could be tricky in this way.
